PodcastGenerator/themes/default/index.php

```diff
@@ -43,6 +43,24 @@
         echo '    <meta property="og:url" content="' . $config["url"] . 'index.php?name=' . $correctepisode["episode"]["filename"] . '" />' . "\n";
         echo '    <meta property="og:image" content="' . $img . '" />' . "\n";
         echo '    <meta property="og:description" content="' . $config["podcast_description"] . '" />' . "\n";
+        if (strtolower($config["enablestreaming"]) == "yes") {
+            // Get mime
+            $mime = getmime($config["absoluteurl"] . $config["upload_dir"] . $correctepisode["episode"]["filename"]);
+            if (!$mime)
+                $mime = null;
```

Comment:
Shouldn't you just nicely exit?

Reply:
I think I did it this way to reduce further nesting. I suppose that I could have put a label after the `if` block and `goto` it instead, but I'm so used to avoiding `goto` except to escape complex program flow situations so I didn't consider it at the time. Let me know if you want me to switch to `goto` and label.